### PR TITLE
Provide README

### PR DESCRIPTION
**Pull request by miri64**
Provides a README with some instructions (and the current build status of the repo).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[![Build Status](https://travis-ci.org/RIOT-OS/applications.svg?branch=master)](https://travis-ci.org/RIOT-OS/applications)
+
+This repository provides some applications for the [RIOT operating system][riot-repo]. Some of them are just useful tools for
```
Comment (smlng) [inline README.md]: lots of `some`s in here, I'd suggest to change the first one and state `provides more applications for the RIOT ...`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+[![Build Status](https://travis-ci.org/RIOT-OS/applications.svg?branch=master)](https://travis-ci.org/RIOT-OS/applications)
+
```
Comment (smlng) [inline README.md]: also add a proper heading here: `# RIOT Applications`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+This repository provides some applications for the [RIOT operating system][riot-repo]. Some of them are just useful tools for
+development work, some showcase software provided with RIOT beyond the simple
+examples within the RIOT codebase.
+
```
Comment (smlng) [inline README.md]: maybe a sub-heading like `## Usage` (or similar)

**Comment by miri64** [comment]
> add some structure?

Done

File: README.md (hunk 1 above)
Comment (miri64) [reply README.md]: Done.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+The submodule to the RIOT repository in this repository will always point to the
+latest release. To change the RIOT version to build against (e.g. current
+master), clone the RIOT repository in a separate repository and point the
+`RIOTBASE` environment variable there:
+
+```sh
+# assuming you are in the working directory of your local clone of this repo
+cd ..
+git clone git@github.com:RIOT-OS/RIOT.git
```
Comment (smlng) [inline README.md]: mhm, wouldn't it be easier to simply go into the submodule path and do `git fetch --all && git checkout <commit>` so no need for an extra clone and (re)setting `RIOTBASE`?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+[![Build Status](https://travis-ci.org/RIOT-OS/applications.svg?branch=master)](https://travis-ci.org/RIOT-OS/applications)
+
+# RIOT Applications
+
+This repository provides more applications for the [RIOT operating system][riot-repo].
+Some of them are just useful tools for development work, others showcase
+software provided with RIOT beyond the simple examples within the RIOT codebase.
+
```
Comment (waehlisch) [inline README.md]: Some of them are useful tools for development work, others showcase more extensive implementations of features in RIOT compared to the simple examples included in the [RIOT main codebase][riot-repo/examples].

File: README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+## Usage
+
+To build and use them follow [the instructions in the RIOT repository][getting-started]
+and the respective application's README.
```
Comment (waehlisch) [inline README.md]: and the READMEs which are part of each application directory.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+The submodule to the RIOT repository in this repository will always point to the
```
Comment (waehlisch) [inline README.md]: In this application repository, the RIOT main code base is included as submodule, which points to the latest RIOT release.

File: README.md (hunk 4 above)
Comment (smlng) [reply README.md]: yeah sure, maybe extend the description to show both options. Like `Either set/update the submodule to the desired commit/release or ...` separate clone.

However, add a note that changing the RIOTBASE or specifically building against another commit than the latest release might not work or show errors, as we do not test against these

**Comment by miri64** [comment]
@smlng @waehlisch are you happy with the newest wording?

**Comment by waehlisch** [comment]
"software provided with RIOT beyond the simple examples [within the RIOT
codebase][riot-repo/examples]." sounds still weird.

**Comment by miri64** [comment]
Oops, I overlooked, that that comment also contained a wording change. I only focused on the reference.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+Alternatively, you can step into the submodule and check out the current master:
+
+```sh
+cd RIOT
+git checkout master
+git pull
+```
+
+Note, that this might lead to build errors, as we do not this repository against
```
Comment (smlng) [inline README.md]: missing word: `... we do not [test] this repository ...`

File: README.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+current master.
```
Comment (smlng) [inline README.md]: well, its not just current master, essentially we only test against the submodule commit, which should be the latest release - older and newer commits might be broken and produce what ever errors.

**Comment by waehlisch** [comment]
> 
> 
> Oops, I overlooked, that that comment also contained a wording change. I only focused on the reference.

that's the reason why I copied more than the reference.

File: README.md (hunk 8 above)
Comment (waehlisch) [reply README.md]: no comma before that

Comment (miri64) [reply README.md]: Fixed

File: README.md (hunk 9 above)
Comment (miri64) [reply README.md]: Is the new wording better?

**Comment by miri64** [comment]
@smlng ping?

**Comment by miri64** [comment]
Wasn't squashed....

**Comment by smlng** [comment]
damn, I'm not used to check this ... we definitely need more checks here. I suggest to cleanup and force-push on master, to me its not a problem here.

**Comment by miri64** [comment]
This would remove the merge commit. Let's keep it as is.

**Pull request merged**